### PR TITLE
rename `appId` to `appName`

### PR DESCRIPTION
**Pull request by pwzgorilla**

### CODE REVIEW

File: src/types/version.go [hunk 1]
```diff
@@ -2,7 +2,7 @@ package types
 
 type Version struct {
 	ID                string            `json:"id,omitempty"`
-	AppID             string            `json:"appID,omitempty"`
+	AppName           string            `json:"name,omitempty"`
```
Comment (xychu) [inline src/types/version.go]: Why not `appName` in json part?

File: src/manager/framework/state/convert.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func AppToRaft(app *App) *rafttypes.Application {
 	return raftApp
 }
 
-func VersionToRaft(version *types.Version, appID string) *rafttypes.Version {
+func VersionToRaft(version *types.Version, appId string) *rafttypes.Version {
```
Comment (upccup) [inline src/manager/framework/state/convert.go]: appID

File: src/manager/framework/state/convert.go [hunk 3]
```diff
@@ -44,7 +44,8 @@ func VersionToRaft(version *types.Version, appID string) *rafttypes.Version {
 		Uris:        version.URIs,
 		Ip:          version.IP,
 		Mode:        version.Mode,
-		AppID:       appID,
+		AppName:     version.AppName,
+		AppID:       appId,
```
Comment (upccup) [inline src/manager/framework/state/convert.go]: appId => appID

File: src/manager/framework/state/convert.go [hunk 4]
```diff
@@ -73,7 +74,7 @@ func VersionToRaft(version *types.Version, appID string) *rafttypes.Version {
 func VersionFromRaft(raftVersion *rafttypes.Version) *types.Version {
 	version := &types.Version{
 		ID:          raftVersion.ID,
-		AppID:       raftVersion.AppID,
```
Comment (upccup) [inline src/manager/framework/state/convert.go]: retain AppID

File: src/manager/raft/types/application.proto [hunk 5]
```diff
@@ -44,9 +44,10 @@ message Version {
     repeated string uris = 16;
     repeated string ip = 17;
     string mode = 18;
-    string appId = 19 [(gogoproto.customname) = "AppID"];
-    int32 priority = 20;
-    repeated string args = 21;
+    string appName = 19 [(gogoproto.customname) = "AppName"];
```
Comment (upccup) [inline src/manager/raft/types/application.proto]: [(gogoproto.customname) = "AppName"]  is unnecessary

**Comment by upccup** [comment]
+1

**Pull request merged**